src/preset_cli/cli/superset/export.py

```diff
@@ -244,16 +271,30 @@ def export_users(ctx: click.core.Context, path: str) -> None:
     type=click.Path(resolve_path=True),
     default="roles.yaml",
 )
+@click.option(
+    "--force-unix-eol",
+    is_flag=True,
+    default=False,
+    help="Force Unix end-of-line characters, otherwise use system default",
+)
 @click.pass_context
-def export_roles(ctx: click.core.Context, path: str) -> None:
+def export_roles(
+    ctx: click.core.Context,
+    path: str,
+    force_unix_eol: bool = False,
+) -> None:
     """
     Export roles to a YAML file.
     """
     auth = ctx.obj["AUTH"]
     url = URL(ctx.obj["INSTANCE"])
     client = SupersetClient(url, auth)
 
-    with open(path, "w", encoding="utf-8") as output:
+    if force_unix_eol:
+        newline = "\n"
+    else:
+        newline = None
```

Comment:
Since this happens in a few different places, another potential approach is to create a small helper function in this file to evaluate the `force_unix_eol` value and return the appropriate `newline` configuration, and then in each of these occurrences you would only call this helper method passing the `force_unix_eol` value.

What are your thoughts?

Reply:
Yes I think a helper method makes sense. Latest changes include this.